frontend/app/craco.config.js

```diff
@@ -113,7 +113,8 @@ module.exports = {
         webpackConfig.output.filename = "static/js/[name].js"
         webpackConfig.output.cssFilename = "static/css/[name].css"
         // Use fixed names for font files as well.
```

Comment:
The comment is invalid. Do we even need to update the field. It's the same as if `OMIT_HASH_FROM_MAIN_FILES` is false, right?

Reply:
Correct. I updated the PR description to reflect that. I'll remove this comment as well.